src/tracks/SampleTrack.cpp

```diff
@@ -556,7 +569,7 @@ void SampleTCOView::paintEvent( QPaintEvent * pe )
 
 		p.setBrush( QBrush( textColor() ) );
 		p.drawEllipse( 4, 5, 4, 4 );
-	}*/
+	}
```

Comment:
These recording indicators can overlay the sample track TCO name, for example: ![image](https://user-images.githubusercontent.com/25503477/36640469-5f7874b6-1a30-11e8-9838-df220d7ce811.png)
How about a different color shade for TCOs as a recording indicator?

Reply:
Also, the current recording indicators can get cut when zooming out in the song editor.
![Rec](https://user-images.githubusercontent.com/25503477/36640319-26f199f4-1a2d-11e8-8da5-dd90887eef54.png)

Reply:
thanks! I think it would be better if we'll just use a red circle and a different background.